rest/config.go

```diff
@@ -1351,6 +1355,39 @@ func (sc *ServerContext) _fetchAndLoadDatabase(nonContextStruct base.NonCancella
 	return true, nil
 }
 
+// migrateV30Configs checks for configs stored in the 3.0 location, and migrates them to the db registry
+func (sc *ServerContext) migrateV30Configs(ctx context.Context) error {
+	groupID := sc.Config.Bootstrap.ConfigGroupID
+	buckets, err := sc.BootstrapContext.Connection.GetConfigBuckets()
+	if err != nil {
+		return err
+	}
+
+	for _, bucketName := range buckets {
+		var dbConfig DatabaseConfig
+		legacyCas, getErr := sc.BootstrapContext.Connection.GetMetadataDocument(bucketName, PersistentConfigKey30(groupID), &dbConfig)
+		if getErr == base.ErrNotFound {
+			continue
+		} else if getErr != nil {
+			return fmt.Errorf("Error retrieving 3.0 config for bucket: %s, groupID: %s: %w", bucketName, groupID, err)
+		}
+
+		_, insertErr := sc.BootstrapContext.InsertConfig(ctx, bucketName, groupID, &dbConfig)
```

Comment:
Should we log at Info before/when doing this?

Reply:
Added logging here.